# Structural bioinformatics of bacterial outer membrane beta-barrel enzymes and their AlphaFold2 predicted water-soluble QTY variants
Beta-barrel enzymes are an important area of study in the field of structural biology.
These proteins serve crucial roles, acting as porins, transporters, enzymes, virulence
factors, and receptors. Recent research has unveiled a novel role for beta-barrel
enzymes in the bacterial outer membrane as sentinels. They remain inactive when the
outer membrane is intact but activate in response to host immune responses and
antibiotics that breach this barrier. Understanding their structure and function is pivotal
in grasping their sentinel role in the bacterial outer membrane. Here we present our
structural bioinformatics analyses on four bacterial outer membrane beta-barrel
enzymes: a) OMPLA, b) OmpT, c) PagP from E.coli, and d) PagL from Pseudomonas
aeruginosa. We previously applied the QTY code to convert hydrophobic alpha-helices
and beta-barrels into hydrophilic forms in over 50 membrane proteins. The QTY code
directly swaps hydrophobic amino acids leucine (L), isoleucine (I), valine (V), and
phenylalanine (F) with hydrophilic glutamine (Q), threonine (T), and tyrosine (Y). We
superposed the structures of native beta-barrel outer membrane enzymes with their
AlphaFold2-predicted QTY variant structures that showed remarkable similarity despite
the replacement of at least 22.95% amino acids in transmembrane regions, the
superposed structures displayed notable structural similarity, indicated by RMSD
values ranging from 0.181Å to 0.286Å. We also analyze the hydrophobicity patches
and the enhanced hydrophilic surfaces. Our research provide insights into the
structural similarity of hydrophobic and hydrophilic beta-barrel enzymes, validating the
utility of the QTY code for investigating beta-barrel membrane enzymes. Our results
not only demonstrate that the QTY code serves as a straightforward tool for designing
water-soluble membrane proteins across various biological contexts, but it may also
stimulate experiments to validate our structural bioinformatic studies.
